### PR TITLE
Add a documentation page for the data catalog

### PR DESCRIPTION
**Pull request by ANich**

### CODE REVIEW

**Review by kwindau — APPROVED** [review]

**Review by scholtzan — APPROVED** [review]
I believe an entry needs to be added to the `SUMMARY.md` so that this page shows up in the table of contents: https://raw.githubusercontent.com/mozilla/data-docs/main/src/SUMMARY.md

File: src/cookbooks/analysis/data_catalog.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+It provides an automatically updated "map" of data assets, including lineage and descriptions, without the need for manual curation.
+
+## What do I use it for?
+
+The primary use case for the catalog is finding out (at a glance) which data assets exist and how they relate to one another. A few examples:
+
+- Finding the source ping or table from a Looker dashboard.
+- Finding out whether a source ping or table has any downstream dependencies.
+- Getting a high-level overview of how tables are transformed before data shows up in a dashboard.
+- Tracing a column through various BigQuery tables.
```
Comment (lucia-vargas-a) [inline src/cookbooks/analysis/data_catalog.md:13]:
```suggestion
- Tracing a column and its transformations through various BigQuery tables.
```

File: src/cookbooks/analysis/data_catalog.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Does this replace tools like the Glean Dictionary or the Looker Data Dictionary?
+
+No. While the features between the Data Catalog and tools such as the Glean Dictionary and Looker Data Dictionary overlap,
+the Data Catalog is meant to be less focused on any single tool and more on assets from all the tools in our data platform,
+providing lineage and reference material that links them together.
+
+## What software does it use?
+
+The catalog is a managed version of open source DataHub - https://datahubproject.io, a metadata platform built and
```
Comment (bani) [inline src/cookbooks/analysis/data_catalog.md:34]: I suggest some of those links could go inline.
Markdown syntax is `[text](url)`

File: src/cookbooks/analysis/data_catalog.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Reference material for data assets (tables, dashboards, pings, etc.) can primarily be found in the Data Catalog: https://mozilla.acryl.io.
+It provides an automatically updated "map" of data assets, including lineage and descriptions, without the need for manual curation.
+
+## What do I use it for?
+
+The primary use case for the catalog is finding out (at a glance) which data assets exist and how they relate to one another. A few examples:
+
+- Finding the source ping or table from a Looker dashboard.
+- Finding out whether a source ping or table has any downstream dependencies.
```
Comment (lucia-vargas-a) [inline src/cookbooks/analysis/data_catalog.md:11]:
```suggestion
- Finding any existing upstream and downstream dependencies for a table, view or source ping.
```

File: src/cookbooks/analysis/data_catalog.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Navigate to https://mozilla.acryl.io and log in via SSO. Once logged in, you should be able to explore assets via the
+search bar or by clicking on a platform (e.g. BigQuery or Glean).
+
+## When was this implemented?
+
+We tested a number of tools in 2022 and finally settled on Acryl. Integration work proceeded from there and continues as
+we add more tools and assets to our data platform.
+
+## Does this replace tools like the Glean Dictionary or the Looker Data Dictionary?
```
Comment (lucia-vargas-a) [inline src/cookbooks/analysis/data_catalog.md:26]:
```suggestion
## Is the Data Catalog a replacement for the Glean Dictionary or the Looker Data Dictionary?
```

File: src/cookbooks/analysis/data_catalog.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+providing lineage and reference material that links them together.
+
+## What software does it use?
+
+The catalog is a managed version of open source DataHub - https://datahubproject.io, a metadata platform built and
+maintained by the company Acryl - https://www.acryldata.io.
+
+## How is the metadata populated?
+
+Metadata is pulled from each platform that is cataloged. Depending on the source, metadata ingestion is either managed
```
Comment (lucia-vargas-a) [inline src/cookbooks/analysis/data_catalog.md:39]: Would be beneficial to elaborate on what being "cataloged" means for a platform.

File: src/cookbooks/analysis/data_catalog.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+- Finding the source ping or table from a Looker dashboard.
+- Finding out whether a source ping or table has any downstream dependencies.
+- Getting a high-level overview of how tables are transformed before data shows up in a dashboard.
+- Tracing a column through various BigQuery tables.
+- Finding the source query or DAG that powers a particular BigQuery table.
+
+## How do I use it?
+
+Navigate to https://mozilla.acryl.io and log in via SSO. Once logged in, you should be able to explore assets via the
```
Comment (lucia-vargas-a) [inline src/cookbooks/analysis/data_catalog.md:18]: Would an image be valuable here?

**Review by lucia-vargas-a — APPROVED** [review]
LGTM, just some suggestions that can improve clarity.

**Review by badboy — CHANGES_REQUESTED** [review]
don't forget to add it to SUMMARY.md or it will not show up in the book.

File: src/cookbooks/analysis/data_catalog.md (hunk 6 above)
Comment (ANich) [reply src/cookbooks/analysis/data_catalog.md:18]: The UI changes fairly frequently which would make this out of date quickly

File: src/cookbooks/analysis/data_catalog.md (hunk 3 above)
Comment (ANich) [reply src/cookbooks/analysis/data_catalog.md:11]: This is in the next example

File: src/cookbooks/analysis/data_catalog.md (hunk 2 above)
Comment (ANich) [reply src/cookbooks/analysis/data_catalog.md]: Updated

File: src/cookbooks/analysis/data_catalog.md (hunk 5 above)
Comment (ANich) [reply src/cookbooks/analysis/data_catalog.md]: I'll remove the term it's probably not necessary

File: src/cookbooks/analysis/data_catalog.md (hunk 4 above)
Comment (ANich) [reply src/cookbooks/analysis/data_catalog.md]: Updated

**Pull request merged**